src/components/specimen/components/TitleBar.tsx

```diff
@@ -41,9 +41,26 @@ const TitleBar = (props: Props) => {
     const specimenActions = [
         { value: 'json', label: 'View JSON' },
         { value: 'sidePanel', label: 'View all Annotations' },
-        { value: 'automatedAnnotations', label: 'Trigger Automated Annotations', isDisabled: !KeycloakService.IsLoggedIn() }
+        { value: 'automatedAnnotations', label: 'Trigger Automated Annotations', isDisabled: !KeycloakService.IsLoggedIn() },
+        { value: 'download', label: 'Download as JSON' }
     ];
 
+    /* Function for downloading a Specimen as JSON */
+    const DownloadSpecimen = () => {
+        /* Parse Specimen object to JSON */
+        const jsonSpecimen = JSON.stringify(specimen);
+
+        /* Create JSON file */
+        const jsonFile = new Blob([jsonSpecimen], { type: "text/plain" });
```

Comment:
Is text/plain correct? or should it be `"application/json"`

Reply:
Both work for me locally, changed it to 'application/json' to be inline with the API